packages/room/README.md

```diff
@@ -0,0 +1,210 @@
+# Room Package
+
+This package is used to work with [inLive Hub and Room API services](https://hub.inlive.app/apidocs/index.html).
+
+## Usage
+
+To use this package, you need to import a `Room` module and initialize it in a global scope where you can export and import the room object returned everywhere in your application.
+
+```js
+import { Room } from '@inlivedev/inlive-js-sdk';
+// Or if you prefer to load only the room module
+import { Room } from '@inlivedev/inlive-js-sdk/dist/room.js';
+
+const room = Room()
+```
+
+### Room object
+
+The room object is the object created when the `Room` module is initialized. It consists methods that relates to the room scope. Some methods that directly interact to the room API can run on both client and server sides.
+
+#### Sample Code
+
+```js
+// create a new room
+const newRoom = await room.createRoom('a new room', 'custom-id');
+
+// get a room data
+const roomData = await room.getRoom(newRoom.data.roomId);
+
+// create a new client
+const client = await room.createClient(roomData.data.roomId);
+
+// create a new peer and automatically open connection to the remote peer
+const peer = room.createPeer(roomData.data.roomId, client.data.clientId);
+
+// listen for a specific room event
+room.on(room.event.STREAM_ADDED, function () {
```

Comment:
stream added should be as internal event because this is only use for set the media source. The stream available is the one that is public event because then the user can opt-in to subscribe to specific user. I think you're refer the stream added here as a stream available event because it is a room event, 

Check this SFU doc for more detail https://github.com/inlivedev/sfu#publish-and-subscribe-tracks

Reply:
Mas @tyohan , before continuing further. I want to clarify that the current `STREAM_ADDED` event is emitted for public use once a local MediaStream is added using `peer.addStream` or when a remote stream is available using on `track` event. The users can listen and get the current local and remote streams available.

If we want to follow the sfu convention, I'm fine with changing the event name into `STREAM_AVAILABLE` event. Also, because the current usage for this event is only for public use, I haven't found any use case to use the `STREAM_ADDED` event for internal use in current implementation. 

I updated the code in [4993650](https://github.com/inlivedev/inlive-js-sdk/pull/66/commits/49936507201cec04845159f4848b51d9a45741da)

Reply:
@faiq-naufal STREAM_ADDED is only emitted to the publisher not public so the publisher can set the source of the track to either media or screen. Once the publisher sets the stream source, then STREAM_AVAILABLE will be emitted to the public. So yes, the STREAM_ADDED event is only for internal use.

Reply:
Mas @tyohan, I updated the code again on [4723501](https://github.com/inlivedev/inlive-js-sdk/pull/66/commits/47235013edae01441c328f3279fa4c9e002dcf82). 

The stream added event uses underscore prefix `_STREAM_ADDED`. When the client adds a stream, the `_STREAM_ADDED` event will be emit to the internal. The internal peer will listen for `_STREAM_ADDED` event and call `_onStreamAdded` handler where the track will be added to the peer connection. Then, the `STREAM_AVAILABLE` event is emitted to public.